sources/subs/CacheMethod/Memcached.php

```diff
@@ -181,10 +191,19 @@ public static function title()
 	 *
 	 * @param array() $config_vars
 	 */
-	public static function settings(&$config_vars)
+	public function settings(&$config_vars)
 	{
 		global $txt;
 
-		$config_vars[] = array('cache_memcached', $txt['cache_memcached'], 'file', 'text', $txt['cache_memcached'], 'cache_memcached', 'force_div_id' => 'memcached_cache_memcached');
+		$var = array(
+			'cache_memcached', $txt['cache_memcached'], 'file',
+			'text', $txt['cache_memcached'], 'cache_memcached',
+			'force_div_id' => 'memcached_cache_memcached',
+		);
+		$serversmList = $this->getServers();
+		if (!empty($serversmList))
+			$var['postinput'] = '<br><br>Added servers:<br><br>' . implode('<br>', $serversmList);
```

Comment:
Hard-coded text. :angel: 
Also, please try to avoid using br's in "source" (i noticed another pair somewhere above I think), otherwise Antechinus will come pay you a visit. :stuck_out_tongue: